templates/test/tr_job_result_details.html.ep

```diff
@@ -0,0 +1,7 @@
+% if ($comments and $comments > 0) {
```

Comment:
I am sorry, I don't want to. I commented on this in the corresponding commit message:
https://github.com/okurz/openQA/commit/c0e258d1e16920393e51a6b091600584bc6e0030
> The 'if ($comment and $comments > 0)' is used to prevent warnings about
undefined if comments are not available at all but also not show notifications
if it is always defined by 0 as used in 'previous.html.ep'.

Don't know a better way except for needing to count the available comments within Jobs.pm#show like in sub overview.